cropper/app/lib/AspectRatio.scala

```diff
@@ -0,0 +1,28 @@
+package lib
+
+import scala.annotation.tailrec
+
+object AspectRatio {
+  case class Ratio(friendly: String, width: Int, height: Int)
+
+  val knownRatios = List(
+    Ratio("5:3", 5, 3),
+    Ratio("2:3", 2, 3),
+    Ratio("16:9", 16, 9)
+  )
+
+  @tailrec
+  def gcd(a: Int, b: Int): Int = if (b == 0) a else gcd(b, a % b)
+
+  def calculate(width: Int, height: Int, tolerance: Int = 3) : Option[Ratio] = {
```

Comment:
yea i thought about that a bit, the problem I saw was that the accuracy of the cropping itself has the same level of looseness at all sizes (from what i could tell) and so as the images get smaller and smaller, the tolerance will be less and less pixels which doesn't really represent what's going on. 

I wasn't sure though, I talked about it briefly with @theefer and we decided to let it ride for now. I'm torn on it though. it should probably be a percentage with a minimum or something like that.